crates/zed/build.rs

```diff
@@ -56,5 +57,11 @@ fn main() {
             "cargo:rustc-link-arg-bins=/MANIFESTINPUT:{}",
             manifest.canonicalize().unwrap().display()
         );
+
+        let res = winresource::WindowsResource::new();
```

Comment:
Is this winresource still needed?

Reply:
Yes. `winresource ` calls `rc.exe` to inject the needed infomations and structures into `zed.exe` for us.

Reply:
Understood, thank you!